app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/audit/WrongGGAccountEvent.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.audit
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class WrongGGAccountEvent(
```

Comment:
this doesn't seem to be triggered from anywhere - the case class is not being used outside of this file

Reply:
it should be audited somewhere around here:

https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/SubscriptionController.scala#L105

and here:

https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/RegistrationController.scala#L217

Reply:
Yes, but we don't have the info we need to audit unless the action filter is changed - should be part of another ticket